.github/workflows/Tests.yaml

```diff
@@ -225,7 +225,7 @@ jobs:
           -D CMAKE_C_COMPILER=clang
           -D CMAKE_CXX_COMPILER=clang++
           -D CMAKE_Fortran_COMPILER=gfortran-9
-          -D CHARM_ROOT=/work/charm_7_0_0/multicore-linux-x86_64-clang
+          -D CHARM_ROOT=/work/charm_7_0_0/multicore-linux-x86_64-gcc
```

Comment:
Use the env var `${CHARM_ROOT}` from the container here and in the other places. You can actually delete this line and the build will use `${CHARM_ROOT}` automatically. (Looking at FindCharm.cmake there might be a bug missing a NOT in STREQUAL)

Reply:
Also, if you move this commit into a separate PR that we can merge first, people can already rebase on this and fewer PRs will get disrupted by the new Docker container.

Reply:
See #4677.